content/blog/2018-09-14-smapr.md

```diff
@@ -0,0 +1,637 @@
+---
+slug: "smapr"
+title: Mapping the 2018 East Africa floods from space with smapr
+package_version: 0.1.2
+authors:
+  - name: Max Joseph
+    url: https://github.com/mbjoseph
+date: 2018-09-14
```

Comment:
change date to 2018-09-25

Reply:
Done! https://github.com/ropensci/roweb2/pull/277/commits/969c944c405ea49113fcd1912fc368ea650d93c1